test/actors.test.js

```diff
@@ -0,0 +1,166 @@
+require('dotenv').config();
+
+const request = require('supertest');
+const app = require('../lib/app');
+const connect = require('../lib/utils/connect');
+const mongoose = require('mongoose');
+const Actor = require('../lib/models/Actor');
+const Film = require('../lib/models/Film');
+const Studio = require('../lib/models/Studio');
+
+describe('actor routes', () => {
+  beforeAll(() => {
+    connect();
+  });
+
+  beforeEach(() => {
+    return mongoose.connection.dropDatabase();
+  });
+
+  afterAll(() => {
+    return mongoose.connection.close();
+  });
+  it('post a new actor', () => {
+    return request(app)
+      .post('/api/v1/actors')
+      .send({ name: 'Billy Zane', dob: 'February 24, 1966', pob: 'Chicago' })
+      .then(res => {
+        expect(res.body).toEqual({
+          _id: expect.any(String),
+          name: 'Billy Zane',
+          dob: expect.any(String),
+          pob: 'Chicago',
+          __v: 0
+        });
+      });
+  });
+  it('get all actors', async() => {
+    const actors = await Actor.create([
+      { name: 'Tim Curry' },
+      { name: 'Michael McKean' },
+      { name: 'Lesley Ann Warren' }
+    ]);
+    return request(app)
+      .get('/api/v1/actors')
+      .then(res => {
+        const actorsJSON = JSON.parse(JSON.stringify(actors));
+        actorsJSON.forEach((actor) => {
+          expect(res.body).toContainEqual({ name: actor.name, _id: actor._id });
+        });
+      });
+  });
+  it('get an actor by id', async() => {
+    const actor = JSON.parse(JSON.stringify(await Actor.create({
+      name: 'Tilda Swinton',
+      dob: 'November 5, 1960',
+      pob: 'London' 
+    })));
+    const studio = JSON.parse(JSON.stringify(await Studio.create({
```

Comment:
How could you refactor this so that these variables can be accessible to multiple it blocks? (i.e. so you wouldn't need to create anything within your it block)